### DISCUSSION TITLE
Change how osu lazer artists sort works

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#24429] [Ideas]
on osu! stable, when you selected "sort by: Artists" the song will sort based name alphabetically including Artists and Beatmaps name but in lazer it just care about Artists name and it doesnt care about beatmaps name at all

Ex:
Stable:
![Stable](https://github.com/ppy/osu/assets/36318840/9a81bc7d-958e-4164-9d33-b7b2cbeefc69)
Lazer:
![Lazer](https://github.com/ppy/osu/assets/36318840/121ce48c-435f-4747-ac2e-11e113ec89b3)

**Comment by peppy** [comment]
This looks weird. The code suggests this should already work. I'm guessing it has since been fixed.
